back/lib/carrierwave_temp_remote.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+class CarrierwaveTempRemote
```

Comment:
Could you add an elaborate comment on top of this class to explain what this does, and why we're doing it? If not, pretty sure the next person would be very confused and have no clue :)